Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
 # Upgrade `setuptools`.
-RUN pip install --upgrade pip setuptools
+RUN pip install --upgrade "pip<25" setuptools
```

Comment:
This might be redundant, since whatever is installed here will be overridden on line 31, i.e., it will only affect which pip is used on line 31.